mappings/net/minecraft/network/message/MessageType.mapping

```diff
@@ -1,27 +1,21 @@
 CLASS net/minecraft/class_2556 net/minecraft/network/message/MessageType
-	COMMENT A message type (also known as "chat type") controls whether to display or narrate
-	COMMENT the messages sent to the clients, and if so, how. Message types are registered
-	COMMENT at {@link net.minecraft.util.registry.BuiltinRegistries#MESSAGE_TYPE}. When
-	COMMENT sending a message, the registry key of the message type can be passed to indicate
+	COMMENT A message type (also known as "chat type") controls how to display or narrate
```

Comment:
Unfortunately, now this really becomes a chat type... Feel free to rename it. Mojang refactors.

Reply:
I think this is fine as-is.